src/features/events/components/ParticipantSummaryCard.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  Box,
+  Button,
+  ClickAwayListener,
+  Paper,
+  Popper,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { Check, Settings } from '@mui/icons-material';
+import { FC, useState } from 'react';
+
+import EventDataModel from 'features/events/models/EventDataModel';
+import messageIds from 'features/events/l10n/messageIds';
+import theme from 'theme';
+import ZUICard from 'zui/ZUICard';
+import ZUINumberChip from 'zui/ZUINumberChip';
+import { Msg, useMessages } from 'core/i18n';
+
+type ParticipantSummaryCardProps = {
+  model: EventDataModel;
+};
+
+const ParticipantSummaryCard: FC<ParticipantSummaryCardProps> = ({ model }) => {
+  const eventData = model.getData().data;
+  const participants = model.getParticipants().data;
+  const respondents = model.getRespondents().data;
+  const messages = useMessages(messageIds);
+  const reqParticipants = eventData?.num_participants_required ?? 0;
+  const availParticipants = participants?.length ?? 0;
+  const remindedParticipants =
+    participants?.filter((p) => p.reminder_sent != null).length ?? 0;
+  const signedParticipants =
+    respondents?.filter((r) => !participants?.some((p) => p.id === r.id))
+      .length ?? 0;
```

Comment:
I'm thinking there could be use for setting up a route on the "frontend API" to get these numbers (req, avail, reminded, signed) directly, because I think they will be needed in more places than this and we could put these calculations in one place and be re-used in many places? But I'm going to leave that to @richardolsson to weigh in on :D

Reply:
I think this is a good idea, but requires some more thinking, so lets revisit that separately and not block this PR for it.

The logic looks correct though, so as a first step I would just move it to a model where it can be reused, and I know @rebecarubio is working on something related and could probably take care of that after this is merged.